docs/pages/eas/metadata/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: EAS Metadata
+sidebar_title: Introduction
+hideTOC: true
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Callout } from '~/ui/components/Callout';
+import { Terminal } from '~/ui/components/Snippet';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+<br />
+
+**EAS Metadata** enables you to automate and maintain your app store presence from the command line.
+
+You need to provide a lot of information to multiple app stores before your users can use your app. This information is often about complex topics that don't apply to your app. You have to start a lengthy review process after providing the information. When the reviewer finds any issues in the information you provided, you need to restart this process.
+
+EAS Metadata uses a [**store.config.json**](./config.md#static-store-config) file to provide information instead of going through multiple forms in the app store dashboards. When it's time to update the app stores, you can push the store config to the app stores.
+
+<Terminal cmd={['$ eas metadata:push']} />
+
+EAS Metadata can also instantly identify known app store restrictions that could trigger a rejection after a length review queue.
+
+When adding the store config file to your repository, you can collaborate with others to prepare the app submission.
```

Comment:
```suggestion
Adding the store config file to your repository enables you to collaborate with other team members to prepare the app submission.
```

Does the changed context fit here? It wasn't clear to me before how just by adding the store config I can collaborate with others to prepare the app submission?

Reply:
In this context, I assumed that you know what a git repo is. The collaboration comes from the part that you can add a file to git + work on it together, as opposed to 1 person going to the app dashboard and filling out all forms. If that makes sense 😄

Reply:
Haha I thought one needed to be part of the organization to collaborate or something like that 😄 (My mind automatically went there).

I still think the changes suggested highlight the advantage a developer gets from it.